### ISSUE TITLE
Pages: ellipsis button is not announced automatically

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12865]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to Pages.
1. Swipe right to navigate through the pages list.
1. Experiment opening the more menu (ellipis button) and then closing it and continuing to move through the list.

**Result:** the ellipsis button is not announced automatically and therefore feels disconnected from the page itself, and if you open the BottomSheet and close it again you lose your place. The ellipsis button is announced as "Button. Possibly more." ([48s](https://cloudup.com/cKtzOuD1KXF))

I'm not sure how exactly the ellipsis button should be read—maybe "Page actions. Button."? cc @etoledom

<img src="https://user-images.githubusercontent.com/1119271/68238456-7c911080-ffc6-11e9-8110-f385e3121be0.png" width="270" alt="pages-ellipsis-button">
<sup>Tested with WP Internal 13.6.20191104 (HockeyApp) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
